website/models.py

```diff
@@ -68,6 +68,56 @@ class Meta:
             ("Retired", "Previous researchers on project."),
         )
 
+class Dataset(models.Model):
+    #unique ids are automatically generated by django for every model instance
+    name = models.CharField(max_length=200, default="")
+
+    def __str__(self):
+        return self.name
+
+class ProjectSelector(models.Model):
+    input_dataset = models.IntegerField(blank=True)
```

Comment:
Most of the 'ints' in the issue description of the models were foreign keys.  Is this how Django models foreign keys?